omnibus/config/projects/puppy.rb

```diff
@@ -108,19 +108,19 @@
 # Linux
 if linux?
   if debian?
-    extra_package_file '/etc/init/datadog-agent6.conf'
-    extra_package_file '/lib/systemd/system/datadog-agent6.service'
+    extra_package_file '/etc/init/stackstate-agent6.conf'
+    extra_package_file '/lib/systemd/system/stackstate-agent6.service'
   end
 
   if redhat? || suse?
-    extra_package_file '/lib/systemd/system/datadog-agent6.service'
+    extra_package_file '/lib/systemd/system/stackstate-agent6.service'
   end
 
   # Example configuration files for the agent and the checks
-  extra_package_file '/etc/dd-agent/datadog.yaml.example'
+  extra_package_file '/etc/sts-agent/datadog.yaml.example'
```

Comment:
`stackstate-agent` if possible

Reply:
this is for compatibility with the old v5 agent

Reply:
done